etc/skydive.yml.default

```diff
@@ -100,16 +100,6 @@ agent:
       # delay in seconds between two metric updates
       # metrics_update: 30
   flow:
-    # Probes used to capture traffic.
```

Comment:
The flow probe section has to be removed from the test config file too in tests/tests.go